# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1410640486 14400
#      Sat Sep 13 16:34:46 2014 -0400
# Node ID 4e5fae88065fda4a3ff2b7b121881eca3b608ebc
# Parent  8a6f87637c1651e28184c57ee5a03a193558d9af
add pan docstring to the manual

* plot.txi: Document pan function.

diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -530,16 +530,18 @@ creates two figures, with the first disp
 the second a cosine wave.  Figure numbers must be positive integers.
 
 @DOCSTRING(figure)
 
 @node Manipulation of Plot Objects
 @subsection Manipulation of Plot Objects
 @cindex plotting, object manipulation
 
+@DOCSTRING(pan)
+
 @DOCSTRING(rotate)
 
 @DOCSTRING(rotate3d)
 
 @DOCSTRING(zoom)
 
 @node Manipulation of Plot Windows
 @subsection Manipulation of Plot Windows
